src/adaptor/ini.rs

```diff
@@ -0,0 +1,502 @@
+
+use nom::{IResult, space, alphanumeric, multispace};
+
+use super::{Adaptor, Config, AdaptorError};
+
+use std::collections::{HashMap, hash_map};
+use std::fmt::Debug;
+use std::io::{Read, Write};
+use std::iter::FromIterator;
+use std::str;
+
+/// The adaptor struct for INI files
+/// Later, this might contain parameters for the myriad INI quirks
+pub struct IniAdaptor {
+    
+}
+
+impl IniAdaptor {
+    /// Constructs a new `IniAdaptor`
+    pub fn new() -> IniAdaptor {
```

Comment:
Yeah, this constructor will eventually take in parameters (or a builder), so we should establish the API now.